recipes/maya/meta.yaml

```diff
@@ -1,45 +0,0 @@
-{% set version = "0.1.5" %}
```

Comment:
You don't need / shouldn't remove other recipes from the recipe dir. conda-forge will skip them.